Dalamud/Interface/Internal/Windows/Data/Widgets/AtkArrayDataBrowserWidget.cs

```diff
@@ -28,148 +46,645 @@ public void Load()
+
+            ImGui.TableNextColumn(); // Text
+            if (!isNull)
+                DrawCopyableText(new ReadOnlySeStringSpan(array->StringArray[i]).ToString(), "Copy text");
+        }
+    }
+
+    private void DrawExtendArrayTab()
+    {
+        using var tab = ImRaii.TabItem("Extend Arrays");
+        if (!tab) return;
+
+        var atkArrayDataHolder = RaptureAtkModule.Instance()->AtkArrayDataHolder;
+
+        this.DrawArrayList(
+            this.extendType,
+            atkArrayDataHolder.ExtendArrayCount,
+            atkArrayDataHolder.ExtendArrayKeys,
+            (AtkArrayData**)atkArrayDataHolder.ExtendArrays,
+            ref this.selectedExtendArray);
+
+        if (this.selectedExtendArray >= atkArrayDataHolder.ExtendArrayCount || atkArrayDataHolder.ExtendArrayKeys[this.selectedExtendArray] == -1)
+            this.selectedExtendArray = 0;
+
+        ImGui.SameLine(0, ImGui.GetStyle().ItemInnerSpacing.X);
+
+        using var child = ImRaii.Child("AtkArrayContent", new Vector2(-1), true, ImGuiWindowFlags.NoScrollbar | ImGuiWindowFlags.NoSavedSettings);
+
+        var array = atkArrayDataHolder.ExtendArrays[this.selectedExtendArray];
+        this.DrawArrayHeader(null, "Extend", this.selectedExtendArray, (AtkArrayData*)array);
+        ImGui.Checkbox("Hide unset entries##HideUnsetExtendArrayEntriesCheckbox", ref this.hideUnsetExtendArrayEntries);
+
+        using var table = ImRaii.Table("ExtendArrayDataTable", 3, ImGuiTableFlags.ScrollY | ImGuiTableFlags.RowBg | ImGuiTableFlags.Borders);
+        if (!table) return;
+
+        ImGui.TableSetupColumn("Index", ImGuiTableColumnFlags.WidthFixed, 40);
+        ImGui.TableSetupColumn("Entry Address", ImGuiTableColumnFlags.WidthFixed, 120);
+        ImGui.TableSetupColumn("Pointer", ImGuiTableColumnFlags.WidthStretch);
+        ImGui.TableSetupScrollFreeze(3, 1);
+        ImGui.TableHeadersRow();
+
+        for (var i = 0; i < array->Size; i++)
+        {
+            var isNull = (nint)array->DataArray[i] == 0;
+            if (isNull && this.hideUnsetExtendArrayEntries)
+                continue;
+
+            using var disabledColor = ImRaii.PushColor(ImGuiCol.Text, ImGui.GetColorU32(ImGuiCol.TextDisabled), isNull);
 
-            ImGui.EndTabBar();
+            ImGui.TableNextRow();
+            ImGui.TableNextColumn(); // Index
+            ImGui.TextUnformatted($"#{i}");
+
+            ImGui.TableNextColumn(); // Address
+            DrawCopyableText($"0x{(nint)(&array->DataArray[i]):X}", "Copy entry address");
+
+            ImGui.TableNextColumn(); // Pointer
+            if (!isNull)
+                DrawCopyableText($"0x{(nint)array->DataArray[i]:X}", "Copy address");
         }
     }
+
+    // Temporarily stored in Dalamud until https://github.com/aers/FFXIVClientStructs/pull/1077 is merged.
+    #pragma warning disable SA1201
+    private enum NumberArrayType
```

Comment:
Please remove and move to CS.

(Yes, I know, we're blocked by CS here.)

Reply:
PR was merged, I removed the enums and checked that it builds (it does).
The CI build here will obviously fail because CS isn't updated.